api/v1alpha1/enterprisecontractpolicy_types.go

```diff
@@ -90,6 +90,24 @@ type EnterpriseContractPolicyExceptions struct {
 	NonBlocking []string `json:"nonBlocking,omitempty"`
 }
 
+// EnterpriseContractPolicyConfiguration configuration of modifications to policy evaluation
+type EnterpriseContractPolicyConfiguration struct {
```

Comment:
I think there's something missing - this needs to be added to the EnterpriseContractPolicySpec somewhere near line 39.

Reply:
FYI @robnester-rh ^

Reply:
Something like this:
```
    Config *EnterpriseContractPolicyConfig `json:"config,omitempty"`,
```

Reply:
I saw the changed CRD yaml file and thought it was included. (@robnester-rh  run `make`) to update the generated CRD yaml. @simonbaird noticed it might need to be run twice for everything to settle…